eras/babbage/formal-spec/utxo.tex

```diff
@@ -46,6 +53,13 @@ \section{UTxO}
         [\var{d}] & \hldiff{(\_, \_, d) \in \var{utxo}~\var{sp},~ \var{d} \in \Datum} \\
         \epsilon  & \text{otherwise}
       \end{cases}
+    \nextdef
+    & \fun{txscripts} : \Tx \to \UTxO \to \powerset{\Script} \\
+    & \fun{txscripts}~tx~utxo = \fun{txwitscripts}~tx \cup \{ \fun{hash}~s \mapsto s \mid (\_, \_, \_, s) \in \var{utxo}~(\fun{spendInputs}~tx \cup \fun{refInputs}~tx)\}
+    \nextdef
+    & \fun{languages} : \Tx \to \UTxO \to \powerset{\Language} \\
+    & \fun{languages}~tx~utxo =
+      \{\fun{language}~s \mid s \in \range (\fun{txscripts}~tx~utxo) \cap \ScriptPhTwo\}
```

Comment:
same as earlier, maybe highlight the new utxo parameter so we can see what changed?